data/core/commands/log.lua

```diff
@@ -0,0 +1,14 @@
+local core = require "core"
+local command = require "core.command"
+
+
+command.add(nil, {
+  ["log:open-as-doc"] = function()
+    local doc = core.open_doc("logs.txt")
+    core.root_view:open_doc(doc)
+    doc:insert(1, 1, core.get_log())
```

Comment:
I suggest adding something like this: 
```lua
doc.new_file = false
doc:clean()
```
Allows you to close the logs and exit the editor without annoying warning about an unsaved file. At the same time, it does not interfere with actually saving as a file